tools/scoring/README.md

```diff
@@ -1,6 +1,11 @@
 # Scoring
 
-The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various _dimensions_.
+The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various [_dimensions_](#dimensions). Broadly, its process is thus:
+
+1. Filter out irrelevant entities (e.g. those which have a noncanonical type)
+2. Isolate the relevant attribute(s) of each entity and reduce them to sets (e.g. [connections](https://github.com/google/digitalbuildings/blob/master/ontology/docs/ontology_config.md#connections))
+3. Count the `intersection` and `difference` between the respective _proposed_ and _solution_ sets
+4. Input these figures to the aforementioned scoring algorithm to produce a singular numeric score
```

Comment:
this is cool ! thank you !
can you please mention how reporting entities vs virtual entities are calculated?

Reply:
https://github.com/google/digitalbuildings/pull/549/commits/af74fa8af4c2ca43b58c7a75686cc04983d4961c